### DISCUSSION TITLE
Welcome to certificate_chains Discussions!

### DISCUSSION THREAD

**Discussion by microbuilder** [microbuilder/certificate_chains#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by d3zd3z** [comment]
This description seems overly general, but I'll go ahead and introduce myself. Maybe it will be a useful record for other that participate later. I'm from the Linaro security working group, and also the current security architect for the Zephyr project. I hope to be able to contribute knowledge on the use of certificates and other related areas for IoT devices.
